packages/tradershub/src/hooks/useRegulationFlags.ts

```diff
@@ -25,7 +25,7 @@ const useRegulationFlags = () => {
         const isNonEURegulation = regulation === Regulation.NonEU;
 
         const isEU = isEUCountry || isEURegulation;
-        const isNonEU = !isEUCountry || isNonEURegulation;
+        const isNonEU = isNonEURegulation;
```

Comment:
The isEUCountry was returning true for nonEU. This check is not needed therefore removed it